test/functional/test-amp-img.js

```diff
@@ -317,7 +279,7 @@ describe('amp-img', () => {
       });
     });
 
-    it('should fallback only once', () => {
+    it('should fallback once and remove fallback once image loads', () => {
```

Comment:
I combined this with the next test because the behavior here changed. Since we no longer set `src` based on the result of `parseSrcset`, the only thing that determines whether an image should show or hide its fallback is whether or not the image loaded, i.e. the result of `loadPromise`.